backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js

```diff
@@ -0,0 +1,29 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const docClient = new dynamodb.DocumentClient();
+const { USER_TABLE } = require('../../constants');
```

Comment:
Can you write unit tests for this lambda please. The don't really need to be that complex, just a case testing that you call the dynamo function with the values passed from event and that you throw an error if sub isn't defined